src/service/standard-actions-impl.js

```diff
@@ -177,6 +177,12 @@ export class StandardActions {
         Services.historyForDoc(this.ampdoc).goBack();
         return null;
 
+      case 'windowBack':
```

Comment:
Are we sure we want to use a new action vs. parameterize `goBack`?

Reply:
Do you have an argument for a parameterized `goBack` vs. `windowBack`? I'd be fine with implementing it that way. Would you want it to be just a boolean or a named parameter?

e.g. `AMP.goBack(window=true)` vs. `AMP.goBack(true)`

Reply:
Having a parameter seems like a less confusing API. I'd suggest `AMP.goBack(navigate=true)`.